src/components/CommentInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import { Comment } from "../types/Types";
+
+type Comments = {
```

Comment:
if later there are not only comments

```suggestion
type Props = {
```